content/english/Kusto-KC7/_index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "KC7: Cyber Defender Game"
+description: "KC7 is a game that teaches you real world cybersecurity skills used by professional Cyber Defenders"
+date: 2023-02-1T13:24:17-07:00
+prereq: "None"
+difficulty: "Advanced"
+draft: false
+icon: "fas fa-database"
+weight: 131
+---
+
+# Introduction: Learn Kusto Query Language (KQL) to Catch the Hacker!
```

Comment:
Let's leave out KQL for now and leave the intro generic

Also, we can't use h1 (#) anywhere, demote all headers one level
```suggestion
## Introduction: Learn Kusto Query Language (KQL) to Catch the Hacker!
```

Reply:
Ok done